guides/common/modules/proc_creating-a-user.adoc

```diff
@@ -4,6 +4,10 @@
 Use this procedure to create a user.
 To use the CLI instead of the {ProjectWebUI}, see the xref:cli-creating-a-user_{context}[].
 
+If a new user is not assigned a default organization, their access is limited.
+To grant systems rights to users, assign them to a default organization.
+The next time the user logs on to {Project}, the user's account has the correct system rights.
```

Comment:
Maybe mention *Default on login* which is responsible to "preselect" an org after login?

Reply:
That's a good idea. Adding a step for that might make lines 7-9 unnecessary, actually. But it would mean larger changes to the procedure for creating a user. Let me try to tackle that in https://github.com/theforeman/foreman-documentation/pull/2865.

For now, I'll mark this thread as resolved because lines 7-9 are not new content, they existed before, just in another module.